base/special/exp10.jl

```diff
@@ -0,0 +1,129 @@
+#  Method
+#  1. Argument reduction: Reduce x to an r so that |r| <= 0.5*log10(2). Given x,
+#     find r and integer k such that
+#
+#                x = k*log10(2) + r,  |r| <= 0.5*log10(2).
+#
+# 2. Approximate exp10(r) by a polynomial on the interval [-0.5*log10(2), 0.5*log10(2)]:
+#
+#           exp10(x) = 1.0 + polynomial(x),
+#
+#    sup norm error within the interval specified, of the polynomial approximations:
+#    Float64 : [2.7245504724394698951963418186308011736517648015976e-18; 2.72455298957534767204462079073786522870226850867e-18]
+#    Float32 : [9.602647147784220587121706280697708280058577656746e-10; 9.6026560194009888671163788405817409356838220899077e-10]
+#
+# 3. Scale back: exp10(x) = 2^k * exp10(r)
+
+# log2(10)
+const LOG210 = 3.321928094887362347870319429489390175864831393024580612054756395815934776608624
+# log10(2)
+const LOG102 = 3.010299956639811952137388947244930267681898814621085413104274611271081892744238e-01
```

Comment:
These (and the horner macro) seem like suspiciously many digits, when they will have to get truncated internally when they are read in (e.g. this one will become 0.3010299956639812). Is it typical to over-state the accuracy in this way?